runtimes/main/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Eliott Teissonniere <git.eliott@teissonniere.org>"]
 edition = "2018"
-name = "runtime-main"
+name = "main-runtime"
```

Comment:
why change it? :)

Reply:
Good one, restored back.
Indeed its my bad, while implementing below chain identification api,
I was using `starts_with()`, so changed the name of crate.

```
fn is_runtime_staking(&self) -> bool {
	self.id().to_lowercase().starts_with("staking")
}
```

But finally id is picked from chain spec definition ...

```
ChainSpec::from_genesis(
	"Staking Local Testnet",
	"staking_local_testnet",
	ChainType::Local,
	local_staking_genesis,
	vec![],
	None,
	Some("nodl"),
	Some(build_local_properties()),
	Default::default(),
)
```